### ISSUE TITLE
Activities in vim can't sync to wakatime server due to python ImportError.

### ISSUE THREAD

**Issue by iaGuoZhi** [wakatime/vim-wakatime#139]
I have used vim-wakatime for several monthes. However today I found my code activities can not sync to dashboard. When I run wakatime binary file in my shell, the following python ImportError occurs:
<img width="793" alt="image" src="https://user-images.githubusercontent.com/41318515/164978169-fe1872a1-d98f-40f2-8007-89dfb100b8eb.png">

This problem appeared after I updated my Manjaro packages through pacman this afternoon. Maybe there are some newer version packages that wakatime doesn't support?

Here is my environments: 

OS: Manjaro 21.2.6
VIM: VIM 8.2
Python: 3.10.4

Thanks!

**Comment by alanhamlett** [comment]
Try `pip uninstall wakatime`?

**Comment by iaGuoZhi** [comment]
Thanks a lot!

After `pip uninstall wakatime` and reinstall vim-wakatime plugin. vim-wakatime finally works. And actully I installed wakatime through pip to enable wakatime in zsh previously.

Before I reinstall vim-wakatime plugin, I found that my .wakatime/wakatime-cli binary file is empty, I thought it may because when /usr/bin/wakatime is installed by pip, vim-wakatime will use it directly but not install .wakatime/wakatime-cli on my machine?
